recipes/openpmix/build.sh

```diff
@@ -0,0 +1,17 @@
+set -ex
+
+# touch generated files to prevent unnecessary invocation of automake
+touch Makefile.in configure
+
+# show available configure options
+# ./configure --help
```

Comment:
```suggestion
```
This commented out code appears only be needed for debugging purpose and should be removed.